js/tabs.js

```diff
@@ -352,20 +348,27 @@ Tabs.prototype.promptSave_ = function(tab, callbackShowDialog) {
   this.dialogController_.show(callbackShowDialog);
 };
 
-Tabs.prototype.save = function(opt_tab, opt_close) {
+/**
+ * Save opt_tab, or the current tab if no opt_tab is passed.
+ * @param {Tab=} opt_tab
```

Comment:
To be clear: you can call this function as

tabs.save();  // or
tabs.save(myTab);  // or
tabs.save(myTab, () => { /* do something here */ });

Is myTab non-nullable? is tabs.save(null); something you'd like to permit? If so, ?Tab, else !Tab

Reply:
My question about nullability applies to other Tab= types, but not to function()= as function() is non-nullable my default.

Reply:
Done.